src/components/menu-list/menu-list-renderer.tsx

```diff
@@ -17,6 +18,15 @@ export class MenuListRenderer {
 
     private applyTabIndexToItemAtIndex: number;
 
+    public componentDidLoad(item) {
+        if (item.iconColor) {
+            /* eslint-disable-next-line no-console */
+            console.warn(
+                "The `iconColor` prop is deprecated now! Use the new `Icon` interface and instead of `iconColor: 'color-name'` write `icon {name: 'icon-name', color: 'color-name'}`."
+            );
+        }
+    }
```

Comment:
@adrianschmidt I cannot figure out how to do it for menu-list and list. Can you add a suggestion or fixup plz?

Reply:
I don't have time right now. I can look at it during cooldown if you remind me. I think it's ok to leave those two for now though…

Or maybe someone else can find the time to help you? Someone working on the pothole cycle maybe? 🤞 🤗

Reply:
found a way